src/Payum/Core/CoreGatewayFactory.php

```diff
@@ -202,7 +202,7 @@ protected function buildClosures(ArrayObject $config)
         }
 
         foreach ($config as $name => $value) {
-            if (is_callable($value)) {
+            if (is_callable($value) && !(is_string($value) && function_exists('\\'.$value))) {
```

Comment:
This specifically checks only for a global function name. So explicitly configuring a function inside a namespace is most likely intended and should still be executed.